src/utils/types.ts

```diff
@@ -81,39 +81,25 @@ export class RESTAPI extends APIBase {
     protected contentType:string;
     protected acceptType:string;
 
-    get = async (baseurl?:string, contentType?:string, acceptType?:string):Promise<RequestResponseData> => {
+    get = async (baseurl?:string):Promise<RequestResponseData> => {
         let ep:string = baseurl ? baseurl : this.baseurl;
 
-        // Set contentType if exists
-        if(contentType) {
-            this.contentType = contentType;
+        let headers:object = { "Content-Type": this.contentType };
+        if(this.acceptType) {
```

Comment:
You'll need to set an intermediate variable. 
let acceptStr = this.acceptType;
If (acceptType){
acceptStr = acceptType
}